app/src/common/urls.js

```diff
@@ -215,7 +215,6 @@ export const URLS = {
       ...createFilterQuery(filterEntities),
     })}`,
 
-  apiInfo: () => `${urlBase}info`,
   compositeInfo: () => `${compositeBase}info`,
```

Comment:
I suggest to rename `compositeInfo` for example to `appInfo`, it will look more intuitive.

Reply:
Fixed